modules/transcription-service-amberscript/src/main/java/org/opencastproject/transcription/amberscript/AmberscriptTranscriptionService.java

```diff
@@ -271,16 +291,33 @@ public Job startTranscription(String mpId, Track track, String language, String
       jobtype = getAmberscriptJobType();
     }
 
+    int numberOfSpeakers = defaultNumberOfSpeakers;
+    if (track.getMediaPackage() != null) {
+      Set<String> speakers = new HashSet<>();
+      if (speakerMetadataField.equals(SpeakerMetadataField.presenter)
+          || speakerMetadataField.equals(SpeakerMetadataField.both)) {
+        speakers.addAll(Arrays.asList(track.getMediaPackage().getCreators()));
+      }
+      if (speakerMetadataField.equals(SpeakerMetadataField.contributor)
+          || speakerMetadataField.equals(SpeakerMetadataField.both)) {
+        speakers.addAll(Arrays.asList(track.getMediaPackage().getContributors()));
```

Comment:
The methods `getCreators` and `getContributors` are deprecated by now, since they may not necessarily yield accurate results (they will do so most of the time). It's potentially better to use the values from the dublincore catalog.